module/mosh.js

```diff
@@ -933,4 +933,59 @@ async function noShipSelected() {
     //log what was done
     console.log(`Told the user that no character was selected.`);
   });
+}
+
+
+//get item from world or compendiums by id or UUID.
+export async function fromIdUuid(id_uuid, options={}){
+  let type = options.type;
+  let item = await fromUuid(id_uuid,options);
+  if(item != null){
+    //we found the item with the id, it probably was an uuid.
+    return item;
+  }
+
+  //we need to manualy find the item:
+  let currentLocation = '';
+  let objectLocation = '';
+  //loop through each compendium
+  game.packs.forEach(function(pack){ 
+    //is this a pack of rolltables?
+    if (pack.metadata.type === type) {
+      //log where we are
+      currentLocation = pack.metadata.id;
+      //loop through each pack to find the right table
+      pack.index.forEach(function(table) { 
```

Comment:
Why the parameter is called `table`, isn't the function generic to find any kind of item? 
It looks confusing to me.

Reply:
it just from the copy-paste from actor.js xD ill change it

Reply:
fixed in the last commit. :)